packages/console/src/pages/TenantSettings/Subscription/index.tsx

```diff
@@ -23,14 +25,27 @@ function Subscription() {
 
   const reservedSkus = pickupFeaturedLogtoSkus(logtoSkus);
 
-  const { data: periodicUsage, isLoading } = useSWR(
-    isCloud && `/api/tenants/${currentTenantId}/subscription/periodic-usage`,
-    async () =>
-      cloudApi.get(`/api/tenants/:tenantId/subscription/periodic-usage`, {
-        params: { tenantId: currentTenantId },
-      })
+  const { data: periodicUsage, error: periodicUsageError } = useSWR<
+    NewSubscriptionPeriodicUsage,
+    ResponseError
+  >(isCloud && `/api/tenants/${currentTenantId}/subscription/periodic-usage`, async () =>
+    cloudApi.get(`/api/tenants/:tenantId/subscription/periodic-usage`, {
+      params: { tenantId: currentTenantId },
+    })
   );
 
+  const { data: usageAddOnSkus, error: usageAddOnSkusError } = useSWR<
+    TenantUsageAddOnSkus,
+    ResponseError
+  >(isCloud && isDevFeaturesEnabled && `/api/tenants/${currentTenantId}/add-on-skus`, async () =>
+    cloudApi.get(`/api/tenants/:tenantId/subscription/add-on-skus`, {
+      params: { tenantId: currentTenantId },
+    })
+  );
+
+  const isLoading =
+    (!periodicUsage && !periodicUsageError) || (!usageAddOnSkus && !usageAddOnSkusError);
```

Comment:
nit: The `isLoading` status can be destructured from the `useSwr` hook now.

Reply:
discussed offline